risc0/build/src/lib.rs

```diff
@@ -500,6 +505,9 @@ pub struct GuestOptions {
     /// Features for cargo to build the guest with.
     pub features: Vec<String>,
 
+    /// Binary names to build.
+    pub bin: Vec<String>,
+
```

Comment:
What do we need this for?

Reply:
Oh, I changed this so we would be able to compile a single binary using docker, like specifying the `--bin` flag. (it would break the compilation step otherwise). 

But since today's meeting we discussed about not needing it, I think I'll just revert the commit that introduced changes to `risc0-build`.

Reply:
[Reverted](https://github.com/risc0/risc0/pull/1302/commits/cff8bcd7f98728256955e8e340475f65ae238d05)